src/isar/mission_planner/echo_planner.py

```diff
@@ -89,6 +88,21 @@ def _mission_plan(self, mission_id: int) -> dict:
 
         return response.json()
 
+    def _create_inspection_tasks_from_sensor_types(
+        self, tag_name: str, sensors: List[str]
+    ) -> List[Union[TakeImage, TakeThermalImage]]:
+        measurement_tasks: List[Union[TakeImage, TakeThermalImage]] = []
+        for sensor in sensors:
+            if sensor == EchoInspectionNames.TakeImage.value:
+                measurement_tasks.append(self._create_image_task(tag_name=tag_name))
+            elif sensor == EchoInspectionNames.TakeThermalImage.value:
```

Comment:
Making the EchoInspectionNames a dataclass; one can remove the .value attributes

```suggestion
            if sensor == EchoInspectionNames.TakeImage:
                measurement_tasks.append(self._create_image_task(tag_name=tag_name))
            elif sensor == EchoInspectionNames.TakeThermalImage:
```

Reply:
Changed it to use a function for the mapping of tasks